pages/landing/patcher/contact/_form.html

```diff
@@ -16,9 +16,47 @@
   <input type="hidden" name="hello_gruntwork" value=""/>
   <input type="hidden" name="_landing-page" value="{{ site.url }}{{ page.url }}" />
   <input type="text" name="_gotcha" style="display:none" />
-  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jon Doe" %}
-  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jon@acme.com" type="email" %}
+  {% include_relative _input-text.html id="contact-name" label="Name" required=true placeholder="Jane Doe" %}
+  {% include_relative _input-text.html id="contact-email" label="Email" required=true placeholder="jane@acme.com" type="email" %}
   {% include_relative _input-text.html id="contact-company" label="Company" required=true placeholder="Acme, Inc." %}
+  <div class="form-group">
+    <div class="row">
+      <div class="col-xs-12 col-md-4">
+        <label>Which programming languages are you interested in?</label>
+      </div>
+      <div class="col-xs-12 col-md-4">
+        {% include_relative _input-checkbox.html name="interested-language" id="terraform" value="Terraform" %}
```

Comment:
NIT: there's a list of supported languages/tools in `page.supported`. You could loop over that and generate these dynamically to keep the lists consistent.

Reply:
I am a bit confused on this one, I am trying to update but the support list is at  `patcher/index.html` and this form is at `patcher/contact/_form.html`, do you know how to use these values somewhere else?

Reply:
Ah, right, I forgot these are separate root pages. To extract it to where you can use multiple root pages, you have to pull the data out into a YAML file in the `data` folder.

Reply:
Looks like @marinalimeira removed this section so I'll resolve this thread.